apps/docs/src/common/pages/patterns/stop-a-service-timing-out.tsx

```diff
@@ -47,6 +47,10 @@ const Page: FC<PageProps> = ({ location }) => (
         {
           href: '/patterns/stop-a-service-timing-out',
           text: 'Stop a service timing out'
+        },
+        {
+          href: '/patterns/search-for-pattern',
```

Comment:
pattern => something

Reply:
```suggestion
          href: '/patterns/search-for-something',
```
Just discovered this was a thing! (Thanks @guermanc.)